frontend/pages/SoftwarePage/components/forms/SoftwareAndroidForm/_styles.scss

```diff
@@ -0,0 +1,112 @@
+.software-android-form {
+  position: relative;
+
+  &__overlay {
+    position: absolute;
+    height: 100%;
+    width: 100%;
+    z-index: 1000;
+  }
+
+  &__content-wrapper {
+    @include vertical-form-layout;
+  }
+
+  &__content-disabled {
+    transition: opacity 150ms ease-in-out;
+    opacity: 0.5; // this adds a disabled effect to the modal content
+  }
+
+  &__form-content {
+    @include vertical-form-layout;
+
+    &--disabled {
+      @include disabled;
+    }
+  }
+
+  &__form-fields {
+    display: flex;
+    flex-direction: column;
+    gap: $pad-medium;
+  }
+
+  &__form-frame {
+    display: flex;
+    align-items: flex-start;
+    flex-direction: column;
+    gap: 20px;
+    align-self: stretch;
+
+    .card {
+      display: flex;
+      align-items: flex-start;
+
+      gap: 12px;
```

Comment:
can we use the padding variables in this file for the gaps. If we arent using those variables in the design i'd just use the padding variable closest to it.